plugins/modules/elastic_pkgpolicy_info.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+# Copyright 2021 Expedient
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ansible.module_utils.basic import _ANSIBLE_ARGS, AnsibleModule
+#from ansible.module_utils.basic import *
+
+import json
+  
+try:
+  from ansible_collections.expedient.elastic.plugins.module_utils.ece import ECE
+except:
+  import sys
+  import os
+  util_path = new_path = f'{os.getcwd()}/plugins/module_utils'
+  sys.path.append(util_path)
+  from ece import ECE
+
+try:
+  from ansible_collections.expedient.elastic.plugins.module_utils.kibana import Kibana
+except:
+  import sys
+  import os
+  util_path = new_path = f'{os.getcwd()}/plugins/module_utils'
+  sys.path.append(util_path)
+  from kibana import Kibana
+
+results = {}
+
+def main():
+
+    module_args=dict(   
+        host=dict(type='str',Required=True),
+        port=dict(type='int', default=9243),
+        username=dict(type='str', Required=True),
+        password=dict(type='str', no_log=True, Required=True),   
+        verify_ssl_cert=dict(type='bool', default=True),
+        agent_policy_name=dict(type='str'),
+        agent_policy_id=dict(type='str'),
+        integration_name=dict(type='str', Required=True),
+        pkg_policy_name=dict(type='str', Required=True)
+    )
+    argument_dependencies = []
+        #('state', 'present', ('enabled', 'alert_type', 'conditions', 'actions')),
+        #('alert-type', 'metrics_threshold', ('conditions'))
+    
+    module = AnsibleModule(argument_spec=module_args, required_if=argument_dependencies, supports_check_mode=True)
```

Comment:
Since a name or id is required as an input, the following logic would ensure the requirement is met. 

```
module = AnsibleModule(argument_spec=module_args, supports_check_mode=True,
                           mutually_exclusive=[('agent_policy_name', 'agent_policy_id')],
                           required_one_of=[('agent_policy_name', 'agent_policy_id')])
```